backend/routes/index.js

```diff
@@ -0,0 +1,85 @@
+var express = require('express');
+var router = express.Router();
+var { generateToken, sendToken } = require('../utils/token.utils');
+var passport = require('passport');
+var config = require('../config');
+var request = require('request');
+require('../passport')();
+
+router.route('/auth/twitter/reverse')
```

Comment:
If all the routes have the prefix /auth, you could add it the app.js file and remove it from here. See the service route for example
Also you can mention the route as the first argument of the .post method I guess